### PR TITLE
Small fixes

### LINKED ISSUE #3498: rasa_nlu duckling extractor: incorrect answer deserialization
**Rasa version**: rasa_nlu 0.15.0

**Python version**: 3.6.7

**Operating system** (windows, osx, ...): Ubuntu 18.04.2 LTS

**Issue**:
Answer from duckling is incorrectly deserialized.
The culprit is the method `DucklingHTTPExtractor._duckling_parse` and that in this method 
`return simplejson.loads(response.text)`
is used instead of
`return response.json()`

If extracted entities contain bodies with accents, incorrect deserialization is done by
the method simplejson.loads. The following script isolates and shows the problem:
```
import requests
import logging
from rasa_nlu.utils import configure_colored_logging
import simplejson


def main():
    url = 'http://localhost:5000/parse'
    payload = {
        'text': "9 metrů",
        'locale': 'cs_CS',
        'tz': None,
        'reftime': 1558015069000
    }
    headers = {"Content-Type": "application/x-www-form-urlencoded; charset=UTF-8"}
    response = requests.post(url, data=payload, headers=headers)
    d_1 = simplejson.loads(response.text)
    d_2 = response.json()
    print(d_1)
    print(d_2)
    if d_1 == d_2:
        print('EQUAL')
    else:
        print('NOT EQUAL')


if __name__ == '__main__':
    configure_colored_logging(logging.DEBUG)
    main()

```

The solution should be to use `response.json()` instead of `simplejson.loads` in the above-mentioned method `DucklingHTTPExtractor._duckling_parse`.

**Content of configuration file (config.yml)**:
```yml
irrelevant
```

**Content of domain file (domain.yml)** (if used & relevant):
```yaml
irrelevant
```

### LINKED ISSUE #3519: rasa shell nlu does not correctly pick up intent format
`rasa shell`:
```
Your input ->  /request_restaurant
2019-05-17 17:17:52 DEBUG    rasa.core.processor  - Received user message '/request_restaurant' with intent '{'name': 'request_restaurant', 'confidence': 1.0}' and entities '[]'

Your input ->  /request_restaurant{"cuisine": "spanish"}
2019-05-17 17:22:05 DEBUG    rasa.core.processor  - Received user message '/request_restaurant{"cuisine": "spanish"}' with intent '{'name': 'request_restaurant', 'confidence': 1.0}' and entities '[{'entity': 'cuisine', 'start': 19, 'end': 41, 'value': 'spanish'}]'
```

`rasa shell nlu`:
```
/request_restaurant
{
  "intent": {
    "name": null,
    "confidence": 0.0
  },
  "entities": [],
  "intent_ranking": [],
  "text": "/request_restaurant"
}

Next message:
/request_restaurant{"cuisine": "spanish"}
{
  "intent": {
    "name": "inform",
    "confidence": 0.9612323045730591
  },
  "entities": [
    {
      "start": 32,
      "end": 39,
      "value": "spanish",
      "entity": "cuisine",
      "confidence": 0.9477547000039636,
      "extractor": "CRFEntityExtractor"
    }
  ],
```

### PR DESCRIPTION
**Pull request by erohmensing**
**Proposed changes**:
- fix #3519 
- fix #3498 
- remove bot utterance logging
- update pyproject.toml

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa_nlu#code-style) for instructions)

### CODE REVIEW

File: rasa/nlu/run.py [hunk 1]
```diff
@@ -1,22 +1,32 @@
...
 
     print_success("NLU model loaded. Type a message and press enter to parse it.")
     while True:
-        text = input().strip()
-        r = interpreter.parse(text)
-        print (json.dumps(r, indent=2))
         print_success("Next message:")
+        message = input()
+        if message.startswith(INTENT_MESSAGE_PREFIX):
+            text = message.rstrip()
```
Comment (tmbo) [inline rasa/nlu/run.py]: why do we not use `strip` for both cases?

Comment (erohmensing) [reply rasa/nlu/run.py]: I think I realized this mistake right after requesting the review and leaving the office 😅fixes

**Comment by erohmensing** [comment]
@tmbo fixed the input, would appreciate if you would quick check the pyproject.toml that I just changed too, it was mentioned in the slack channel but looks like no one got around to changing it

**Comment by tmbo** [comment]
Yes looks good :+1: thanks a lot :Star:

**Pull request merged**